go/felix/intdataplane/policy_mgr.go

```diff
@@ -47,13 +49,16 @@ func (m *policyManager) OnUpdate(msg interface{}) {
 	case *proto.ActivePolicyUpdate:
 		log.WithField("id", msg.Id).Debug("Updating policy chains")
 		chains := m.ruleRenderer.PolicyToIptablesChains(msg.Id, msg.Policy, m.ipVersion)
+		m.rawTable.UpdateChains(chains)
```

Comment:
Yes, that was the quick fix I described last night.  Need to revisit to work out if there's a better way...